app/export.go

```diff
@@ -30,7 +30,7 @@ func (app *App) ExportAppStateAndValidators(
 		app.prepForZeroHeightGenesis(ctx, jailAllowedAddrs)
 	}
 
-	genState := app.mm.ExportGenesis(ctx, app.appCodec)
+	genState := app.mm.ExportGenesisForModules(ctx, app.appCodec, modulesToExport)
```

Comment:
Nice that they added this change, this was exactly the functionality I wanted to implement for our export process , this would allow us to not export state data which we don't need to for our test . 
I think for a basic test  you could compare the exported genesis to an existing one , they should contain similar data , a older genesis data would be available at sample.GenDoc , if you rebase with develop. Not sure if we need to be writing tests for this though the ExportCMD is part of the SDK